### PR TITLE
Support for osx-arm64 builds

### PR DESCRIPTION
**Pull request by stepanhruda**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)

Taking over automatic bot PR #3 and adding changes to hopefully get builds working. Looks like the new changes only failed on linux and osx-arm64 builds just fine, so it's probably a result of not re-rendering the config for a very long time rather than related to the new arch.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ build:
 requirements:
   build:
     - cmake >=3.12
+    - make
```
Comment (stepanhruda) [inline recipe/meta.yaml:20]: This is literally the only manual change, everything else is the work of conda-forge's automatic migrator

Comment (ngam) [reply recipe/meta.yaml:20]: @stepanhruda pls ping @tstenner and @conda-forge/core to merge

**Comment by hmaarrfk** [comment]
Do you want to be a maintainer? If so, add yourself and rerender

**Comment by hmaarrfk** [comment]
if not  ping us again ti merge

**Pull request merged**